netbox/extras/management/commands/housekeeping.py

```diff
@@ -81,7 +81,7 @@ def handle(self, *args, **options):
                         ending=""
                     )
                     self.stdout.flush()
-                JobResult.objects.filter(created__lt=cutoff)._raw_delete(using=DEFAULT_DB_ALIAS)
+                JobResult.objects.filter(created__lt=cutoff).delete(using=DEFAULT_DB_ALIAS)
```

Comment:
Should be fine to omit `using` here; it's only needed for the raw method. (Should also delete the import above.)
```suggestion
                JobResult.objects.filter(created__lt=cutoff).delete()
```

Reply:
Removed the using parameter.

> Should also delete the import above

Could you elaborate, not sure I follow.

Reply:
Sorry, I mean the the import of `DEFAULT_DB_ALIAS` can be removed.

Reply:
It is still being used for the cleanup of ObjectChange's, so I'll let it stay for now at least. Not sure how _raw_delete actually functions, but if it doesn't respect cascade deletions, maybe you want to use the nomal delete method for ObjectChanges as well to ensure search indexes are deleted? Just a thought, I'm gonna resolve this for now.